json_container/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 find_package(Boost 1.54 REQUIRED COMPONENTS regex)
+find_package(rapidjson 1.1.0 REQUIRED)
```

Comment:
The file is named `RapidJSON` upstream:
https://github.com/Tencent/rapidjson/blob/master/RapidJSON.pc.in

This seems to be causing the failures…